probe-test-build/build.rs

```diff
@@ -17,16 +17,6 @@ use usdt::Builder;
 fn main() {
     println!("cargo:rerun-if-changed=build.rs");
 
-    if !version_check::is_min_version("1.59").unwrap_or(false) {
```

Comment:
Can we now remove `version_check` entirely, if we don't need it here?

Reply:
From `build-dependencies`? Yeah, good point -- done.